ERROR:root:РћС€РёР±РєР° РІ РёР·РјРµРЅРµРЅРёРё С‚РµРєСѓС‰РµР№ РґРёСЂРµРєС‚РѕСЂРёРё!
WARNING:root:РћС€РёР±РєР° РІ РёР·РјРµРЅРµРЅРёРё С‚РµРєСѓС‰РµР№ РґРёСЂРµРєС‚РѕСЂРёРё!
WARNING:root:РћС€РёР±РєР° РІ РёР·РјРµРЅРµРЅРёРё С‚РµРєСѓС‰РµР№ РґРёСЂРµРєС‚РѕСЂРёРё!
WARNING:root:РћС€РёР±РєР° РІ РІС‹РІРѕРґРµ СЃРѕРґРµСЂР¶РёРјРѕРіРѕ С„Р°Р№Р»Р° testFile!
utils.py[LINE:72]# WARNING  [2021-05-28 12:23:07,564]  Ошибка в создании папки!
utils.py[LINE:73]# INFO     [2021-05-28 21:11:29,224]  Пользователь qwe создал папку test в директории: docs/qwe//
